hexvexed-0.0.1.recipe

```diff
@@ -1,6 +1,7 @@
-SUMMARY="A hexagon based game for Haiku"
-DESCRIPTION="This is a hexagon based tile puzzle game forked from DarkWyrm's BeVexed \
-code to support 6 sides instead of four."
+SUMMARY="A tile puzzle game for Haiku"
+DESCRIPTION="This is a great puzzle game for Haiku,Verry addictive game and easy to play.\
+Just put hexagon tiles in the grid on left such that the numbers match whatever 2 tiles touch. \
+This game forked from DaekWyrm's BeVexed but you can arrange 6 sides instread of 4."
 
```

Comment:
This gets embarrassing... there are still grammatical/typo errors both from vanish and me...:

```
DESCRIPTION="HexVexed is a maddeningly addictive and easy to play puzzle game:
Arrange hexagonal tiles into a grid so that numbers and colours match \
where they touch. This game is a version DarkWyrm's BeVexed, but instead \
of squares, the tiles are hexagonal shaped. It's 1.5x the fun!